### PR TITLE
docs: use /api and /v2/api path for swagger spec files to match node

### PR DESCRIPTION
**Pull request by sborrazas**
refs #108
refs #674

### CODE REVIEW

**Review by jyeshe — COMMENTED** [review]
There is a also `code:priv_dir` that gives the priv directly but no big deal. Missing more the test cases to avoid manual testing.

**Comment by sborrazas** [comment]
> There is also `code:priv_dir` that gives the priv directly but no big deal. Missing more the test cases to avoid manual testing.

Good call on the `priv_dir/1`. Also, I added unit/integration test cases for it

File: Dockerfile [hunk 1]
```diff
@@ -82,15 +82,16 @@ COPY lib lib
...
-
 # Compile the release
 RUN mix compile
 
 # Changes to config/runtime.exs don't require recompiling the code
 COPY config/runtime.exs config/
 
+# Generate swagger V2 file
+RUN mix run --no-start -e 'IO.puts(Mix.Project.config[:version])' >AEMDW_VERSION
+RUN scripts/swagger-docs.py >priv/static/swagger/swagger_v2.yaml
```
Comment (jyeshe) [inline Dockerfile:93]: :clap:

**Review by jyeshe — APPROVED** [review]
:gear:

**Pull request merged**